README.md

```diff
@@ -1,2 +1,17 @@
-# sig-contributor-experience
-TODO
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Contributor Experience SIG
+
+## What is the Contributor Experience SIG?
+
+The goal of this SIG is to improve the experience of contributors to the OpenTelemetry project. 
+To achieve this, we create programs and processes around key contribution stages, such as first commit, or the transition from regular maintainer to approver.  We also consult with other SIGs to feedback into the process with an aim to improve overall community health.
+
+## Get Involved
+
+There are many ways to get involved in our work:
+
+- Create [issues]([github.com/open-telemetry/sig-contributor-experience/issues/](https://github.com/open-telemetry/sig-contributor-experience/issues/new)), comment on [existing issues](github.com/open-telemetry/sig-contributor-experience/issues/), or [review pull requests](https://github.com/open-telemetry/sig-contributor-experience/pulls).
```

Comment:
Consider splitting into three bullet list items.

Reply:
I wanted to avoid having a sub list of items, it looks convoluted in the markdown, because of the links, but the rich view is ok